migrate-from-aurora-using-lightning.md

```diff
@@ -79,12 +79,12 @@ summary: 了解如何使用 TiDB Lightning 从 Amazon Aurora MySQL 迁移全量
 1. 使用 Dumpling 导出表结构文件：
 
     ```
-    ./dumpling --host 127.0.0.1 --port 4000 --user root --password password --no-data --output ./schema --filter "mydb.*"
+    ./dumpling --host database-1.cedtft9htlae.us-west-2.rds.amazonaws.com --port 3306 --user root --password password --consistency none --no-data --output ./schema --filter "mydb.*"
```

Comment:
I think locking these tables for a short while is better.
```suggestion
    ./dumpling --host database-1.cedtft9htlae.us-west-2.rds.amazonaws.com --port 3306 --user root --password password --consistency lock --no-data --output ./schema --filter "mydb.*"
```

Reply:
Tried the command with --consistency lock, it works. To keep doc identify with TiDB Cloud https://docs.pingcap.com/tidbcloud/public-preview/migrate-from-aurora-bulk-import, shall we keep none?